component/src/main/java/com/celements/blog/plugin/BlogClasses.java

```diff
@@ -285,11 +346,21 @@ BaseClass getNewsletterConfigClass() throws XWikiException {
     return bclass;
   }
 
+  /**
+   * @deprecated instead use {@link BlogArticleSubscriptionClass}
+   * @since 6.5
+   */
+  @Deprecated(since = "6.5", forRemoval = true)
   public DocumentReference getBlogArticleSubscriptionClassRef(String wikiName) {
     return new DocumentReference(wikiName, BLOG_ARTICLE_SUBSCRIPTION_CLASS_SPACE,
         BLOG_ARTICLE_SUBSCRIPTION_CLASS_DOC);
   }
 
+  /**
+   * @deprecated instead use {@link BlogArticleSubscriptionClass}
+   * @since 6.5
+   */
+  @Deprecated(since = "6.5", forRemoval = true)
```

Comment:
just fyi, no change needed: deprecating the class already deprecates all methods/properties.